cluster/gce/gci/configure-helper.sh

```diff
@@ -409,6 +409,23 @@ function assemble-docker-flags {
   fi
 
   echo "DOCKER_OPTS=\"${docker_opts} ${EXTRA_DOCKER_OPTS:-}\"" > /etc/default/docker
+
+  # If using a network plugin, extend the docker configuration to always remove
+  # the network checkpoint to avoid corrupt checkpoints.
+  # (https://github.com/docker/docker/issues/18283).
+  if [[ "${use_net_plugin}" == "true" ]]; then
```

Comment:
I don't fully understand the network plugin check. Why do we want to do this only in this case?
Even the case below. You would have to restart docker daemon for any flag change to take effect. Not just the network plugin one.